source/text/label.ts

```diff
@@ -266,7 +285,11 @@ export class Label {
      * texture atlas).
```

Comment:
wrong doc: lineWidth is *not* expected in typesetting space unit, but in label.fontSizeUnit